packages/http/src/parameters.ts

```diff
@@ -15,13 +15,16 @@ import {
   HttpOperationParameters,
   HttpVerb,
   OperationParameterOptions,
+  PathParameterOptions,
+  QueryParameterOptions,
 } from "./types.js";
+import { parseUriTemplate } from "./uri-template.js";
 
 export function getOperationParameters(
   program: Program,
   operation: Operation,
+  partialUriTemplate: string,
```

Comment:
It looks like you've updated `@typespec/rest` and that only the first 2 params are used in all invocations from the typespec-azure repo - anywhere else we might need to check if this is breaking?

Reply:
I need to make a PR against typespec-azure to check breaking change but this one I checked and its not used anywhere else. It was a little too much internal function anyone actually consuming the library should use `getHttpOperation`